config/initializers/curation_concerns_actors.rb

```diff
@@ -0,0 +1,2 @@
+CurationConcerns::Actors::ActorFactory.remove_actor CurationConcerns::Actors::AddToCollectionActor
```

Comment:
Yes.

I don't really care that master is broken right now honestly. I just want a common starting point for planning, which I now have.